### PR TITLE
[4.0] Missing language strings

### PR DESCRIPTION
**Pull request by astridx**
### Summary of Changes
Add of missing language strings

### Testing Instructions
1. Try to install a manifests with a api section but missing api files. If you like, you can take this zip:
[com_foos.zip](https://github.com/joomla/joomla-cms/files/4407168/com_foos.zip)
2. You see this message
Extensions  Install   test   Administration
3. Apply this patch and redo the installation with the same manifest.
You see the same message, but the language string is translated.
Extensions  Install   test   Administration(1)

### CODE REVIEW

File: administrator/language/en-GB/lib_joomla.ini [hunk 1]
```diff
@@ -530,6 +530,7 @@ JLIB_INSTALLER_ABORT_COMP_BUILDADMINMENUS_FAILED="Error building Administrator M
 JLIB_INSTALLER_ABORT_COMP_COPY_MANIFEST="Component %1$s: Could not copy PHP manifest file."
 JLIB_INSTALLER_ABORT_COMP_COPY_SETUP="Component %1$s: Could not copy setup file."
 JLIB_INSTALLER_ABORT_COMP_FAIL_ADMIN_FILES="Component %s: Failed to copy administrator files."
+JLIB_INSTALLER_ABORT_COMP_FAIL_API_FILES="Component %s: Failed to copy api files."
```
Comment (brianteeman) [inline administrator/language/en-GB/lib_joomla.ini]:
```suggestion
JLIB_INSTALLER_ABORT_COMP_FAIL_API_FILES="Component %s: Failed to copy API files."
```

File: language/en-GB/lib_joomla.ini [hunk 2]
```diff
@@ -530,6 +530,7 @@ JLIB_INSTALLER_ABORT_COMP_BUILDADMINMENUS_FAILED="Error building Administrator M
 JLIB_INSTALLER_ABORT_COMP_COPY_MANIFEST="Component %1$s: Could not copy PHP manifest file."
 JLIB_INSTALLER_ABORT_COMP_COPY_SETUP="Component %1$s: Could not copy setup file."
 JLIB_INSTALLER_ABORT_COMP_FAIL_ADMIN_FILES="Component %s: Failed to copy administrator files."
+JLIB_INSTALLER_ABORT_COMP_FAIL_API_FILES="Component %s: Failed to copy api files."
```
Comment (brianteeman) [inline language/en-GB/lib_joomla.ini]:
```suggestion
JLIB_INSTALLER_ABORT_COMP_FAIL_API_FILES="Component %s: Failed to copy API files."
```

**Comment by astridx** [comment]
@brianteeman Thank you. I corrected the string.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 4087b2ea7389ccf2ddb83995fbe3eaa850375659<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28519">issues.joomla.org/tracker/joomla-cms/28519</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 4087b2ea7389ccf2ddb83995fbe3eaa850375659<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28519">issues.joomla.org/tracker/joomla-cms/28519</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28519">issues.joomla.org/tracker/joomla-cms/28519</a>.</sub>

**Comment by infograf768** [comment]
tks

**Pull request merged**